### DISCUSSION TITLE
Radeon GPU's

### DISCUSSION THREAD

**Discussion by timboxyz** [neonbjb/tortoise-tts#143] [Q&A]
I am wondering why only Nvidia GPU's and if this is ever likely to change?

**Comment by neonbjb** [comment]
Hey there,
They work fine, I just don't have any documentation available because I don't have any Radeon gpus to try. Prs welcome if you want to help out.

**Reply by timboxyz** [reply]
Oh right. I was wondering if it specifically needed CUDA since that seems to be PyTorch's preferred GPU install type.

On PyTorch's git there are a number of reported issues of AMD GPU usage performing slower than CPU based calls.

I will give it a try but am not that hopeful. Will let you know how it goes.

**Comment by jnordberg** [comment]
Can confirm it runs fine on PyTorch with ROCm 5.1.1 with no performance issues on a rx6900

**Reply by timboxyz** [reply]
When I looked at the options for installing PyTorch on my Mac everything (CUDA & ROCm) except "default" was crossed through. I couldn't discover what "default" meant except that it seems to be different to "CPU only". I suspect it means limited GPU acceleration available. Vega 20 GPU.

**Comment by neonbjb** [comment]
Unfortunately that doesn't surprise me. :(

**Reply by timboxyz** [reply]
I am told (via PyTorch forums) that default probably includes MPS. However, I also note outstanding bug reports that running on AMD GPU can be slower than using CPU on a Mac where ROCm seems unavailable. I will continue to investigate this on PyTorch discussions. Thanks for all the input that has helped me know where to look.
